src/components/generic/InputField/InputField.scss

```diff
@@ -5,6 +5,12 @@ $C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-input-field;
   flex-direction: column;
   position: relative;
 
+  pointer-events: none;
+
+  & * {
```

Comment:
use the direct child selector please, `& > *`, because the '*' applies to every element in the page, and the browser will only look at the parent to see if it's a .wxc-input-field instead of looking the whole dom up.

Reply:
and actually you can use just `> *`, the `& ` prefix is always implied